### DISCUSSION TITLE
WiFi Usage on Seeed Studio XIAO ESP32C3 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#65] [Q&A]
# WiFi Usage on Seeed Studio XIAO ESP32C3 | Seeed Studio Wiki

WiFi Usage on Seeed Studio XIAO ESP32C3

https://wiki.seeedstudio.com/XIAO_ESP32C3_WiFi_Usage/

**Comment by BartelMan** [comment]
Esp32c3 board: it is stated that: ⚠️ Please be careful when you try to use the board as a hotspot(access point). There may be a overheat problem and cause burns.

can you elaborate on that? What conditions? What counter measures?
Thanks,

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.
